.github/workflows/canary.yaml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - 'main'
+    paths:
+      - '**.go'
```

Comment:
I think we should also build it when updating `Dockerfile.canary` and `.github/workflows`.

Reply:
Done